torchinductor/utils.py

```diff
@@ -52,3 +55,23 @@ def sympy_dot(seq1, seq2):
 
 def unique(it):
     return {id(x): x for x in it}.values()
+
+
+def timed(model, example_inputs, times=1):
+    synchronize()
+    torch.manual_seed(1337)
+    t0 = time.perf_counter()
+    for _ in range(times):
+        result = model(*example_inputs)
+        synchronize()
```

Comment:
Should we synchronize after all runs are done?

Reply:
Also, we could also perhaps just use triton's benchmarking code.

Reply:
It is useful to measure CPU overheads.  Plus this works for the C++ backend.  We use the gpu-only measurement in other places though.